code_of_conduct.md

```diff
@@ -0,0 +1,34 @@
+<h2>Code Of Conduct-CYBERSENTINEL</h2>
```

Comment:
I'm unable to make sense of the word CYBERSENTINEL. The rest looks good.

Reply:
change in file section @bhaskar1001101